### ISSUE TITLE
Better alternative to fix PyQT5.Sip error

### ISSUE THREAD

**Issue by Geek-ubaid** [GDGVIT/pip-gui#22]
## Issue Description

While installing the pipgui(locally) package the following steps were taken

### Executed action
The following commands were executed serially
```bash
$ virtualenv --python=python3 venv
$ source venv/bin/activate
$ python setup.py build
$ python setup.py install
$ pipgui
```

### Outcome

![image](https://user-images.githubusercontent.com/30529572/72897125-320e6d00-3d47-11ea-8c07-026658fc4110.png)

The same thing was happening when installing through pip

```bash 
$ pip install pipgui
```

## How the issue was fixed

The PyQt5.sip error fixed by fixing the dependency of the following packages in the install_require setting param of setup.py. 

The existing install_require now includes packages with the following dependencies :
- PyQt5==5.10.1
- sip==4.19.8

This is a temporary fix for now. Searching for an efficient solution for the bug.

## Things that did'nt worked

Tried to fix the errors in the following ways
- [Method1](https://coldfix.de/2019/03/14/no-pip-no-sip/)
- [Method2](https://stackoverflow.com/a/42967199) 

## Why the current solution is not good
The method used to solve the error is mentioned [here](https://stackoverflow.com/questions/51098152/installing-pyqt5-from-sources-unable-to-import-pyqt5-sip)
The above method used is not a **long term solution** as the mentioned version of the packages may get **depreciated** in the near future. 

## Current requirement

Better alternative for fixing the PyQt5.Sip error

**Comment by r00tdaemon** [comment]
Is this still valid?
Having no issues with latest pyqt5

**Comment by Geek-ubaid** [comment]
@ujjwal96 I will verify the installation before closing the issue

**Comment by Geek-ubaid** [comment]
@ujjwal96 the fix in the [PR](#29 ) resolved this issue. This issue no longer exists now.
